doc/rtd/explanation/configuration.rst

```diff
@@ -79,3 +83,4 @@ images.
 
 .. _Config: https://github.com/canonical/cloud-init/blob/b861ea8a5e1fd0eb33096f60f54eeff42d80d3bd/cloudinit/settings.py#L22
 .. _cloud.cfg template: https://github.com/canonical/cloud-init/blob/main/config/cloud.cfg.tmpl
+.. _YAML 1.1: https://yaml.org/spec/1.1/current.html
```

Comment:
```suggestion
.. _YAML version 1.1: https://yaml.org/spec/1.1/current.html
```

Reply:
The mismatch is (I think) what's throwing the spellcheck error. Even though this isn't an error in the spelling.